evaluator.py

```diff
@@ -187,17 +265,28 @@ def calculate_gbps_http(self, result: dict) -> Bitrate:
         return -1  # type: ignore
 
     def evaluate_pass_fail_status(self) -> PassFailStatus:
-        total_passing = 0
-        total_failing = 0
+        tft_passing = 0
+        tft_failing = 0
         for result in self.test_results:
             if result.success:
-                total_passing += 1
+                tft_passing += 1
             else:
-                total_failing += 1
+                tft_failing += 1
+
+        plugin_passing = 0
+        plugin_failing = 0
+        for plugin_result in self.plugin_results:
+            if plugin_result.success:
+                plugin_passing += 1
+            else:
+                plugin_failing += 1
+
         return PassFailStatus(
-            result=total_failing == 0,
-            num_passed=total_passing,
-            num_failed=total_failing,
+            result=tft_failing + plugin_failing == 0,
```

Comment:
```🜛 ~ python3
Python 3.12.3 (main, Apr  9 2024, 08:09:14) [Clang 15.0.0 (clang-1500.3.9.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> True + 1
2
```
In python booleans can be added to integers so let's make sure we doing the addition first (would make sense to add parenthesis in this case)

Reply:
There is operator precedence in python so this will not happen:

https://www.programiz.com/python-programming/precedence-associativity#:~:text=The%20operator%20precedence%20in%20Python%20is%20listed%20in%20the%20following%20table.%20It%20is%20in%20descending%20order%20(upper%20group%20has%20higher%20precedence%20than%20the%20lower%20ones).

Also, these are not booleans, they are integers.